src/client.h

```diff
@@ -542,11 +545,21 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	// Send the item number 'item' as player item to the server
 	void sendPlayerItem(u16 item);
 
+	void deleteAuthData();
+	// helper method shared with clientpackethandler
+	static AuthMechanism get_auth_mech(const u32 mechs);
+
 	void sendLegacyInit(const char* playerName, const char* playerPassword);
+	void sendInit(const std::string &playerName);
+	void startAuth(AuthMechanism chosen_auth_mechanism);
 	void sendDeletedBlocks(std::vector<v3s16> &blocks);
 	void sendGotBlocks(v3s16 block);
 	void sendRemovedSounds(std::vector<s32> &soundList);
 
+	// Helper function
+	inline std::string getName()
+	{ return m_env.getLocalPlayer()->getName(); }
```

Comment:
Shouldn't this be something more like getLocalPlayerName to be more descriptive??